frameworks/hdfs/tests/test_overlay.py

```diff
@@ -8,23 +8,24 @@
 import sdk_networks as networks
 
 
+@pytest.fixture(scope='module', autouse=True)
+def configure_package(configure_universe):
+    try:
+        install.uninstall(PACKAGE_NAME)
+        utils.gc_frameworks()
+        install.install(PACKAGE_NAME, DEFAULT_TASK_COUNT,
+                        additional_options=networks.ENABLE_VIRTUAL_NETWORKS_OPTIONS)
 
+        yield # let the test session execute
+    finally:
+        install.uninstall(PACKAGE_NAME)
 
-def setup_module(module):
-    install.uninstall(PACKAGE_NAME)
-    utils.gc_frameworks()
-    install.install(PACKAGE_NAME, DEFAULT_TASK_COUNT,
-                    additional_options=networks.ENABLE_VIRTUAL_NETWORKS_OPTIONS)
 
-
-def setup_function(function):
+@pytest.fixture(autouse=True)
```

Comment:
I didn't know that. Cool. I do like having the decorator attached to the function though. I think it's a bit clearer to the next person reading the code.